spacewalk/certs-tools/mgr_ssl_cert_setup.py

```diff
@@ -0,0 +1,465 @@
+#!/usr/bin/python3
+#
+# Copyright (c) 2021, SUSE LLC
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+
+## language imports
+import os
+import sys
+import argparse
+import traceback
+import shutil
+import tempfile
+import time
+import subprocess
+from datetime import datetime
+
+from spacewalk.common.rhnLog import initLOG, log_time, log_clean
+from uyuni.common.fileutils import getUidGid
+
+logfile = '/var/log/rhn/mgr-ssl-cert-setup.log'
```

Comment:
Small question: isn't this a constant? Is there any reason to not define this as `LOGFILE` as the rest of constants?
```suggestion
LOGFILE = '/var/log/rhn/mgr-ssl-cert-setup.log'
```